msteams-platform/tabs/how-to/conversational-tabs.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Create conversational tabs
+author: laujan
+description: Create conversational sub-entity chat for your channel tabs
+keywords: teams tabs channel configurable 
+ms.topic: conceptual
+ms.author: lomeybur
+---
+# Create conversational tabs
+
+Conversational sub-entities provides a way to allow your users to have conversations about (sub-)entities in your tab (such as specific task, patient, sales opportunity, etc) instead of discussing the entire tab (aka: entity). A traditional channel/configurable tab allows the user to have a conversation about a tab, but your users may not find this sufficient. Either because there is too much content to have a centralized discussion or because the content changes over time, making the conversation no longer relevant to the content being shown. Conversational sub-entities provides a much more focused conversation experience for dynamic tabs. 
+
+Conversational sub-entities are only supported in channels. However, they can be used from a personal/static tab to create (or continue) conversations in tabs that are _already_ pinned to a channel. The latter is useful if you wish to provide one location for a user to view and access conversations happening across multiple channels.
+
+## Requirements
+
+In order to support conversational sub-entities, your tab web application will need the ability to store a mapping between sub-entities ↔ conversations in a backend database. We will provide you with the `conversationId`, but it will be your responsibility to store that `conversationId` and return it to Teams in order for users to continue the conversation.
```

Comment:
Question:
Shall we add the following information as a Note?

We will provide you with the `conversationId`, but it will be your responsibility to store that `conversationId` and return it to Teams in order for users to continue the conversation.